sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/ViewBuilder.java

```diff
@@ -73,7 +82,8 @@ public ViewBuilder setAggregation(Aggregation aggregation) {
    */
   public ViewBuilder setAttributeFilter(Predicate<String> keyFilter) {
     Objects.requireNonNull(keyFilter, "keyFilter");
-    return addAttributesProcessor(AttributesProcessor.filterByKeyName(keyFilter));
```

Comment:
Can the `addAttributesProcessor` method be removed? It does not seem to be used anywhere

Reply:
It gets called reflectively by `SdkMeterProviderUtil`, which provides experimental methods for adding attributes processors which add keys from baggage. 

The change here is worth calling out. I've changed the behavior of `setAttributeFilter` from chaining together to setting the attribute filter, erasing any previous. I believe this is the correct, and is consistent with out places where we have the opportunity to merge or override in a builder. For example, in the `Sdk{Signal}ProviderBuilder`s, we have `setResource`, which sets the resource, overriding any previously set, and `addResource`, which merges with the existing. 

If generic attribute filters become part of our public API some day, we can consider having both `set*` and `add*` variants.